Rust/tokio-http/README.md

```diff
@@ -8,6 +8,8 @@ example in the tokio repository.
 
 Be sure to update your rust stable toolchain to at least 1.60.0.
 
+Please ensure that you're using the standard toolchain (rustup default).
```

Comment:
I'm not sure what this is for, there's sentence right above stating that stable toolchain should be used

Reply:
Mike asked me to emphasize this.

Reply:
If you want to emphasize this then I would propose merging this line and the prior line into:

> Be sure to use a stable Rust toolchain that is no older than Rust 1.60.0. You can use `cargo --version` to view the version of your default Rust toolchain.